lib/config.js

```diff
@@ -130,9 +130,9 @@ function config (opts) {
   opts = Object.assign(
     {},
     DEFAULTS, // default options
-    readEnv(), // options read from environment variables
     confFile, // options read from elastic-apm-node.js config file
-    opts // options passed in to agent.start()
+    opts, // options passed in to agent.start()
+    readEnv() // options read from environment variables
```

Comment:
@felixbarny just added you as reviewer as well as I wanted to confirm that this is the same order that you have implemented in the Java agent (with `Object.assign` later arguments override earlier)

Reply:
The order in the Java agent is defaults, env, config file

Reply:
So yes, it's pretty much the same in the java agent

Reply:
java system properties are command line flags right?

Reply:
correct